src/Jackett.Common/Indexers/AniLibria.cs

```diff
@@ -0,0 +1,189 @@
+            baseRelease.BannerUrl = new Uri(resultDocument.QuerySelector("img#adminPoster").Attributes["src"].Value);
+            baseRelease.Comments = uri;
+            baseRelease.DownloadVolumeFactor = 0;
+            baseRelease.UploadVolumeFactor = 0;
+            baseRelease.Category = new int[]{ 1 };
+
+            IEnumerable<IElement> rows = resultDocument.QuerySelectorAll("table#publicTorrentTable > tbody > tr");
+            var releases = new List<ReleaseInfo>();
+            foreach (var r in rows) {
+                var release = (ReleaseInfo)baseRelease.Clone();
+                release.Title += " - " + r.QuerySelector("td.torrentcol1").Text();
+
+                var info = Regex.Replace(r.QuerySelector("td.torrentcol2").Text().TrimStart(' '), @"\s+", " ").Split(' ');
+                release.Size = ReleaseInfo.GetBytes(info[0] + info[1]);
+                release.Seeders = long.Parse(info[2]);
+                release.Peers = long.Parse(info[2]) + long.Parse(info[3]);
+                release.Grabs = long.Parse(info[4]);
+                
+                var dateParts = r.QuerySelector("td.torrentcol3").Text().Split(' ');
+                release.PublishDate = DateTime.ParseExact(dateParts[1] + " " + dateParts[2], "dd.MM.yyyy HH:mm", CultureInfo.InvariantCulture);
+
+                release.Link = GetAbsoluteUrl(r.QuerySelector("td.torrentcol4 > a").Attributes["href"].Value);
+                releases.Add(release);
+            }           
+            
+            return releases;
+        }
+
+        private IEnumerable<ReleaseInfo> ParseResult(string response)
+        {
+            dynamic json = JObject.Parse(response);
+            const string rowSelector = "a";
+
+            try
+            {
+                var resultParser = new HtmlParser();
+                var resultDocument = resultParser.ParseDocument(response);
+                IEnumerable<IElement> rows = resultDocument.QuerySelectorAll(rowSelector);
+
+                var releases = new List<ReleaseInfo>();
+                foreach (var r in rows)
+                    try
+                    {
+                        var releasePageUri = GetAbsoluteUrl(r.Attributes["href"].Value);
+                        releases.AddRange(QueryReleasePage(releasePageUri));
+                    }
+                    catch (Exception ex)
+                    {
+                        logger.Error($"AniLibria: Error parsing search result row '{r.ToHtmlPretty()}':\n\n{ex}");
+                    }
+
+                return releases;
+            }
+            catch (Exception ex)
+            {
+                throw new IOException($"Error parsing search result page: {ex}");
+            }
+        }
+
+        private static string ParseStringValueFromHtml(IHtmlDocument document, string propertyName, string selector)
```

Comment:
I thin you can remove this method. It's used only once and you don't need to handle this exception. If it fails the stacktrace is enough.